transformers/avayaVoiceChannel/README.MD

```diff
@@ -0,0 +1,9 @@
+# Description
+Avaya Voice Channel Transformer translates the communication between Avaya CPaaS calls,sms messages and Cognigy's text in real time.
+
+# Usage
+The first step is to upload Avaya's Extension, then create a Cognigy Flow Bot with Avaya's extension nodes such as play,prompt,record,handover,hangup,conference.
```

Comment:
Please add a detailed description of how to use the extension with this transformer. The developer needs to be able to follow a step-by-step tutorial.

Reply:
made changes per suggested